deployment/config.yaml

```diff
@@ -23,10 +23,10 @@ security:
 
 api:
     specs:
-        - path: storage_handler.yaml
+        - path: api/specs/specs.yaml
           append: null
           add_operation_fields:
-              x-openapi-router-controller: controllers
+              x-openapi-router-controller: tus_storagehandler.api.elixircloud.csh.controllers 

```

Comment:
Not sure if it matters, but just to be on the safe side, consider removing the trailing whitespace that was, apparently, added in one of the latest commits.